<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>系统</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>661</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>状态显示</string>
   </property>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>641</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>实时状态</string>
    </property>
    <widget class="QLabel" name="label_444">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>人脸状态</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>未识别到</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_447">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>是否瞌睡</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>正常</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4410">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>30</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>帧数</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>30</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>60</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_441">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>60</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>眼部状态</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_448">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>嘴巴状态</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>60</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_445">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>头部状态</string>
     </property>
    </widget>
    <widget class="QLabel" name="label446">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>点头次数</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>90</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>90</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_442">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>90</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>眨眼次数</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_449">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>哈欠次数</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>641</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>提示</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPlainTextEdit" name="plainTextEdit_tip"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>0</y>
     <width>191</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>设置</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="6" column="1">
     <widget class="QSpinBox" name="spinBox_3"/>
    </item>
    <item row="7" column="1">
     <widget class="QSpinBox" name="spinBox_4"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_241">
      <property name="text">
       <string>头侧阈值</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_251">
      <property name="text">
       <string>哈欠阈值10s</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QSpinBox" name="spinBox_5"/>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="spinBox_1"/>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBox_21">
      <property name="text">
       <string>头部功能</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="checkBox_32">
      <property name="text">
       <string>弹窗警告</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="checkBox_22">
      <property name="text">
       <string>信息提示</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="checkBox_31">
      <property name="text">
       <string>语音播报</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="checkBox_12">
      <property name="text">
       <string>哈欠识别</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_211">
      <property name="text">
       <string>眨眼阈值10s</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBox_11">
      <property name="text">
       <string>眼部功能</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>开始识别</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="spinBox_2"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_221">
      <property name="text">
       <string>闭眼阈值10s</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_231">
      <property name="text">
       <string>点头阈值10s</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>451</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>图像信息</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_img">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
